pkg/server/util/util.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
😱

This is the beginning of a utils nightmare. Can we leave it in an existing package? Even `pkg/server/requestinfo` is better than a generic utils package.

Reply:
I don't have a strong feeling.

Reply:
Done